README.md

```diff
@@ -32,45 +34,66 @@ allprojects {
 }
 ```
 
-project build.gradle
+project **build.gradle**
 ```groovy
 dependencies {
     commonMainApi("dev.icerock.moko:permissions:0.1.0")
 }
 ```
 
-settings.gradle  
+**settings.gradle**
 ```groovy
 enableFeaturePreview("GRADLE_METADATA")
 ```
 
-## Usage
-TODO
+## List of supported permissions
 
+The full list can be found in `dev.icerock.moko.permissions.Permission` enum.
+
+* Camera: **Permission.CAMERA**
+* Gallery: **Permission.GALLERY**
+* Storage: **Permission.STORAGE**
+* Fine location: **Permission.LOCATION**
+* Coarse location: **Permission.COARSE_LOCATION**
+
+## Usage example
+
+Common code:
 ```kotlin
 class ViewModel(val permissionsController: PermissionsController): ViewModel() {
     fun onPhotoPressed() {
-        launch {
+        coroutineScope.launch {
             try {
                 permissionsController.providePermission(Permission.GALLERY)
-                // granted
-            } catch(error: Throwable) {
-                // denied
+                // Permission has been granted successfully.
+            } catch(deniedAlways: DeniedAlwaysException) {
+                // Permission is always denied (android specific).
```

Comment:
incorrect. AlwaysDenied exist in both platforms. but just Denied is exist only on android. If user denied permission on ios it will be alwaysdenied and we can't re-show system permission dialog

Reply:
The incorrect comment has been deleted.